src/scripts/ccget

```diff
@@ -1,14 +1,9 @@
-#!/usr/bin/env python3
```

Comment:
Shouldn't these not be removed, but replaced with `#!/usr/bin/env python`? If there's no shebang, I don't think the shell will know how to execute the script.